apps/mylar3/config.json

```diff
@@ -1,15 +1,15 @@
 {
   "$schema": "../schema.json",
-  "name": "Mylar3",
+  "name": "Mylar (3)",
```

Comment:
I think this should stay `Mylar3` as it is literally the name of the app (based on python3)